posts/2025-02-17_theres_a_pharmaverse_package_for_that/managing-clinical-trial-data.qmd

```diff
@@ -0,0 +1,329 @@
+
+-   Key players in pharmaverse and whether you need all packages
+
+-   How pharmaverse compares to Tidyverse and how to learn it
+
+By the end, you'll have a clear understanding of how pharmaverse supports clinical trial operations and how to apply these tools in your work.
+
+## Key Stages of Clinical Reporting
+
+Managing clinical trial data involves multiple stages, each with its own challenges. **Pharmaverse** provides a range of R packages that support different parts of the process, sometimes even offering multiple options for the same task. This flexibility allows organizations to choose the best tools for their specific needs rather than sticking to a one-size-fits-all approach.
+
+A **metadata-driven approach** helps ensure that clinical trial data is consistently structured and aligned with regulatory standards. The typical process follows this sequence:
+
+**Metadata** ➝ **OAK** ➝ **Admiral** ➝ **Define.xml** ➝ **TLGs** ➝ **Submissions**
+
+Some examples of **pharmaverse** packages that support clinical reporting include:
+
+-   [**diffdf**](https://cran.r-project.org/web/packages/diffdf/vignettes/diffdf-basic.html) – Tracking differences in datasets.
+-   [**metatools**](https://pharmaverse.github.io/metatools/) – Metadata management and transformation.
+-   [**sdtm.oak**](https://pharmaverse.github.io/sdtm.oak/) – The primary **pharmaverse** package for SDTM dataset creation.
+-   [**sdtmchecks**](https://pharmaverse.github.io/sdtmchecks/) – Validates SDTM datasets for compliance with CDISC standards.
+-   [**admiral**](https://github.com/pharmaverse/admiral) – Standardized data derivations.
+-   [**metacore**](https://cran.r-project.org/web/packages/metacore/index.html) – Metadata-driven structures.
+-   [**rtables**](https://github.com/insightsengineering/rtables) – Structured table outputs.
+-   The **pharmaverse** provides multiple table-making packages, such as [chevron](https://insightsengineering.github.io/chevron/latest-tag/), [Tplyr](https://github.com/atorus-research/Tplyr), [pharmaRTF](https://atorus-research.github.io/pharmaRTF/), [gtsummary](https://www.danieldsjoberg.com/gtsummary/), [cards](https://insightsengineering.github.io/cards/latest-tag/), [tfrmt](https://gsk-biostatistics.github.io/tfrmt/), and [tidytlg](https://pharmaverse.github.io/tidytlg/main/). More tools are listed on the [TLGs page](https://pharmaverse.org/e2eclinical/tlg/).
+-   [**xportr**](https://github.com/atorus-research/xportr) – CDISC-compliant dataset export.
+-   [**pkglite**](https://merck.github.io/pkglite/) – Package management and tracking.
+-   [**metacore**](https://github.com/atorus-research/metacore), [**metatools**](https://pharmaverse.github.io/metatools/) – For standardized metadata structures and validation.
+-   [**sdtmchecks**](https://github.com/pharmaverse/sdtmchecks), [**datacutr**](https://cran.r-project.org/web/packages/datacutr/index.html) – Ensuring data integrity and regulatory compliance.
+
```

Comment:
Can we add https://github.com/pharmaverse/logrx to this list for logging R scripts

Reply:
done